apps/ui/src/composables/useSharing.ts

```diff
@@ -1,14 +1,16 @@
 import { FunctionalComponent } from 'vue';
-import { Proposal, Space, User } from '@/types';
+import { getChoiceText } from '@/helpers/utils';
+import { Choice, Proposal, Space, User } from '@/types';
 import ICFarcaster from '~icons/c/farcaster';
 import ICLens from '~icons/c/lens';
 import ICX from '~icons/c/x';
 
 type SpaceUser = { user: User; space: Space };
 type SocialNetwork = 'x' | 'lens' | 'farcaster';
 
-export type PayloadType = 'proposal' | 'user' | 'space-user';
-export type Payload = User | Proposal | SpaceUser;
+export type Vote = { proposal: Proposal; choice: Choice };
+export type PayloadType = 'proposal' | 'user' | 'space-user' | 'vote';
+export type Payload = Proposal | User | SpaceUser | Vote;
```

Comment:
nit. blank line should be below Vote, not above it, now `Vote` looks like it belongs to `PayloadType` and `Payload` and not `SpaceUser`, `SocialNetwork`.